docs/check-examples

```diff
@@ -62,7 +62,7 @@ defaultCommand=
 defaultExit=
 defaultLang=
 
-while getopts 'd:x:l:' opt; do
+while getopts 'md:x:l:' opt; do
```

Comment:
What does this md means? Can you add a comment about this line?

Reply:
That is from another PR that adds -m as an option. If you don't mind I'll comment it there so I don't have to rerun all the tests again -- the m d x and l are options the : means the preceding option letter takes an argument.